src/textual/actions.py

```diff
@@ -3,6 +3,11 @@
 import ast
 import re
 
+from typing_extensions import Any, TypeAlias
+
+Action: TypeAlias = "tuple[str, tuple[Any, ...]]"
```

Comment:
I'm ambivalent about the name "Action". It's not an action per se, but the result of parsing an encoded action. `"ActionParseResult"` would be more apt, if verbose.

Reply:
I agree.